script/tasks/diamondUpdatePeriphery.sh

```
@@ -186,14 +186,18 @@ register() {
       doNotContinueUnlessGasIsBelowThreshold "$NETWORK"
 
       if [[ "$ENVIRONMENT" == "production" ]]; then
-        # propose registerPeripheryContract transaction to multisig safe
-        local CALLDATA=$(cast calldata "registerPeripheryContract(string,address)" "$CONTRACT_NAME" "$ADDR")
-
-        DIAMOND_ADDRESS=$(getContractAddressFromDeploymentLogs "$NETWORK" "$ENVIRONMENT" "$DIAMOND_CONTRACT_NAME")
+        if [ "$DEPLOY_NEW_NETWORK_MODE" == "true" ]; then
+          echo "DEPLOY_NEW_NETWORK_MODE is activated - registering "$CONTRACT_NAME" as periphery on diamond "$DIAMOND_ADDRESS" in network $NETWORK now..."
```

Shellcheck (0.10.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)